### PR TITLE
Remove relationship

### PR DESCRIPTION
**Pull request by SeemaDeshpande**

### CODE REVIEW

**Comment by rahul-tarafdar-synerzip** [comment]
check if this design is aligned with Sourabh's PR
https://github.com/rahul-tarafdar-synerzip/neo4j-browser/pull/29

**Comment by SeemaDeshpande** [comment]
aligned this pr with Sourabh's changes

File: src/shared/modules/itemEditor/itemEditorDuck.js [hunk 1]
```diff
@@ -120,9 +112,18 @@ export const handleEditEntityEpic = (action$, store) =>
         break
       case 'delete':
         if (action.entityType === 'node') {
-          cmd = `MATCH (p:${action.firstLabel}) where ID(p)=${action.nodeId} OPTIONAL MATCH (p)-[r]-() DELETE r,p`
+          cmd = `MATCH (p:${action.firstLabel}) where ID(p)=${
+            action.nodeId
+          } OPTIONAL MATCH (p)-[r]-() DELETE r,p`
         } else if (action.entityType === 'relationship') {
           // FIXME find out the command for relationship deletion
+        } else if (action.entityType === 'propertyNode') {
```
Comment (rahul-tarafdar-synerzip) [inline src/shared/modules/itemEditor/itemEditorDuck.js]: rename to nodeProperty

File: src/shared/modules/itemEditor/itemEditorDuck.js [hunk 2]
```diff
@@ -120,9 +112,18 @@ export const handleEditEntityEpic = (action$, store) =>
...
+          cmd = `MATCH (a:${action.firstLabel}) where ID(a)=${action.nodeId}
+          REMOVE a.${action.propertyKey} RETURN a, ((a)-->()) , ((a)<--())`
+        } else if (action.entityType === 'propertyRelationship') {
```
Comment (rahul-tarafdar-synerzip) [inline src/shared/modules/itemEditor/itemEditorDuck.js]: rename to relationshipProperty

File: src/shared/modules/itemEditor/itemEditorDuck.js (hunk 1 above)
Comment (SeemaDeshpande) [reply src/shared/modules/itemEditor/itemEditorDuck.js]: Changes done

File: src/shared/modules/itemEditor/itemEditorDuck.js (hunk 2 above)
Comment (SeemaDeshpande) [reply src/shared/modules/itemEditor/itemEditorDuck.js]: Changes done

**Review by rahul-tarafdar-synerzip — APPROVED** [review]

**Pull request merged**